eng/build.ps1

```diff
@@ -27,11 +27,19 @@ Param(
   [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
 )
 
+function Get-Default-Arch() {
+  if ($env:DOTNET_RUNTIME_DEFAULT_ARCH) {
```

Comment:
Happy to entertain suggestions on alternative variable names.

Reply:
This name suggests that this is a broadly applicable default, but it is not the case for the current delta. With the current delta, the name should be something like `DOTNET_RUNTIME_BUILD_PS1_DEFAULT_ARCH`.

Reply:
Updated.